config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  
+  resources :items, only: [:create, :destroy]
```

Comment:
@DarylMcCullough per my Bloc message back to you, I'm fine with keeping it the way it is. Looking forward to your next commit :)